index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <section class="navigation" >
+        <h1 id="name">John Doe</h1>
+        <div id="menu"><a href=""><img src="images/menu.png" alt=""></a></div>
+    </section>
+    <section class="body">
+        <h3 id="one">Hey There.
+            I’m Alex</h3>
+        <div id="two">I’am a Software Developer</div>
+        <div id="three">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</div>
```

Comment:
Instead of using div, we can use a `p` tag for this text, since this is a paragraph and that would make more sense.